buildSrc/src/Tasks.kt

```diff
@@ -178,10 +178,19 @@ open class GenerateChangeLog : ChangeLogTask() {
 
     @TaskAction
     fun generate() {
-        val writer = ChangeLogGenerator(*createWriters())
-        if (includeUnreleased && unreleasedEntries().isNotEmpty()) {
-            writer.unreleased(unreleasedEntries().map { it.toPath() })
+        val writers = createWriters()
+        val writer = ChangeLogGenerator(*writers)
+        logger.info("Generating Changelog of types: ${writers.toList()}")
+        val unreleasedEntries = unreleasedEntries()
+        if (includeUnreleased) {
+            logger.info("Including ${unreleasedEntries.size} unreleased changes")
+            if (unreleasedEntries.isNotEmpty()) {
+                writer.unreleased(unreleasedEntries.map { it.toPath() })
+            }
+        } else {
+            logger.info("Skipping unreleased changes")
```

Comment:
Consider adding `${unreleasedEntries.size}` to the message as a way of knowing how many have built up

Reply:
I dont think it matters, we are opting out of including them, why should we care how many there are?